.github/workflows/security.yml

```diff
@@ -11,10 +11,11 @@ env:
   SNYK_TOKEN: ${{ secrets.CAOS_SNYK_TOKEN }}
   DOCKER_HUB_ID: ${{ secrets.OHAI_DOCKER_HUB_ID }}
   DOCKER_HUB_PASSWORD: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
+  TAG: "0.0.0"
```

Comment:
WDYT if we rename this to DEV_TAG? maybe even move it to env block of the step that is using it. At the first glance it might be confusing